core/src/main/scala/org/apache/spark/shuffle/hash/BlockStoreShuffleFetcher.scala

```diff
@@ -17,22 +17,21 @@
 
 package org.apache.spark.shuffle.hash
 
+import org.apache.spark.storage._
```

Comment:
style nit: import ordering (all of the spark imports should be grouped and alphabetized)